CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 * Ignored hot key command on edit fields. Refs UIIN-2604.
 * Don't render Fast Add record modal in a `<Paneset>` to re-calculate other `<Pane>`'s widths after closing. Fixes UIIN-2690.
 * "Saving instance failed" modal does not show error message. Fixes UIIN-2686.
+* Inactive permanent location for Holdings/items when user open details. Fixes UIIN-2697.
```

Comment:
Looks like it should be `10.0.7`

Reply:
I agree with the original implementation: the target branch here is master, so I think `11.0.0` is the right place right now. If we decide to cherry-pick this on to the `b10.0` branch and publish another patch release, _then_ `10.0.7` will be correct.